harness/determined/_core/_context.py

```diff
@@ -87,10 +89,25 @@ def _dummy_init(
 def init(
     *,
     distributed: Optional[_core.DistributedContext] = None,
+    preempt_mode: _core.PreemptMode = _core.PreemptMode.WorkersAskChief,
     # TODO: figure out a better way to deal with checkpointing in the local training case.
     storage_manager: Optional[storage.StorageManager] = None,
-    preempt_mode: _core.PreemptMode = _core.PreemptMode.WorkersAskChief,
 ) -> Context:
+    """
+    core.init() builds a core.Context for use with the Core API.
+
+    You should always use core.init() instead of instantiating a core.Context directly.  Certain
```

Comment:
What do you mean by "indecisive"?

Also, I thought you told me to say things like "You should do XYZ" instead of "Users should do XYZ".  When does that apply?

Reply:
I tend to prefer imperative sentences when instructing the user to do something. It's more engaging (that's what I meant by indecisive).
You're right about what I said about "you" for all other types of documentation. I don't think "you" should be in a reference doc. But because this includes how-to content, "you" is ok here. Generally, if you can reword it without using "you," it makes a better sentence. But the occasional "you" can give the reader a sense of belonging, which also increases engagement.